### PR TITLE
Add a rule about `send` metaprogramming.

### PR DESCRIPTION
**Pull request by syndbg**
Submitted as requested in https://github.com/bbatsov/rubocop/pull/2081

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3616,6 +3616,9 @@ resource cleanup when possible.
 * <a name="prefer-public-send"></a>
   Prefer `public_send` over `send` so as not to circumvent `private`/`protected` visibility.
 <sup>[[link](#prefer-public-send)]</sup>
+* <a name="prefer-__send__"></a>
```
Comment (bbatsov) [inline README.md]: Add a blank line before the anchor.

File: README.md [hunk 2]
```diff
@@ -3616,6 +3616,9 @@ resource cleanup when possible.
...
+  Prefer `__send__` over `send` as `send` may overlap with existing methods.
```
Comment (bbatsov) [inline README.md]: The rule should be a bit longer - some explanation for it is in order. You should also add a few code snippets to illustrate it.

**Comment by bbatsov** [comment]
And guess you'll submit one more PR about `public_send`, right? :-)

**Comment by syndbg** [comment]
@bbatsov Isn't the rule about `public_send` sufficient? I should add code examples for both `public_send` and `__send__` rules, though.

**Comment by bbatsov** [comment]
Sorry about this. Was a bit distracted when I made my comment. Just remove the `.` from the commit message's title.

**Comment by bbatsov** [comment]
@syndbg ping :-)

**Comment by syndbg** [comment]
@bbatsov Sorry for the delays. Added code examples. 

Turned out that thinking of good examples for trivial stuff is hard. :8ball:

**Comment by syndbg** [comment]
@bbatsov ping.

**Comment by bbatsov** [comment]
:+1:

**Pull request merged**